src/configuration/services/pagespeed.md

```diff
@@ -0,0 +1,41 @@
+# PageSpeed Module
+
+PageSpeed is a family of 4 tools developped by Google Inc to optimize of websites performance.
+
+Pagespeed module, also known as mod_pagespeed, is an open source HTTP module for Apache or Nginx, which runs several optimizations on the resources of your website, on the server side.
+
+See the [PageSpeed docs](https://developers.google.com/speed/) for more information.
+
+## Supported versions
+
+* 1.12
+
+## Setup
+
+In your ``.platform/services.yaml``:
+
+```yaml
+pagespeed:
+    type: pagespeed:1.12-rc
+    configuration:
+        endpoints:
+            app: {}
+    relationships:
+        app: "myapp:http"
+```
+
+The pagespeed proxy exposes an endpoint app and this endpoint uses the upstream relationship named app.
+
+In your .platform/routes.yaml, use pagespeed:http-app instead of app:http:
+
+```yaml
+https://{default}/":
+    type: upstream
+    upstream: "pagespeed:http-app"
```

Comment:
http-app?  Is that the endpoint that Damien gave?  That's... very misleading, as I would expect it to be the HTTP connection to the app container.  Why's it not just `http`, like app containers and the experimental Varnish container?

Reply:
It's `http-<relationship to app>`.

Reply:
So this needs much clearer docs...

Reply:
Yes, so it it correct, but it is a bit confusing.